roles/nvidia-gpu-operator-node-prep/tasks/main.yml

```diff
@@ -7,7 +7,7 @@
   modprobe:
     name: nouveau
     state: absent
-  ignore_errors: true
+  failed_when: false
```

Comment:
Any idea what the functional difference here is?

Reply:
AFAICT, the major difference is that the task isn't marked as `failed` in the final Ansible results summary.

Reply:
From what I can tell the difference is that ignore_errors makes the output red and failed_when doesn't.